addons/website/views/snippets/s_cover.xml

```diff
@@ -2,17 +2,32 @@
 <odoo>
 
 <template id="s_cover" name="Cover">
-    <section class="s_cover parallax s_parallax_is_fixed bg-black-50 pt96 pb96" data-scroll-background-ratio="1">
+    <section class="s_cover parallax s_parallax_is_fixed bg-black-50 o_half_screen_height" data-scroll-background-ratio="1">
         <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0;"/>
         <div class="container s_allow_columns">
-            <h1 style="text-align: center;"><font style="font-size: 62px; font-weight: bold;">Catchy Headline</font></h1>
-            <p class="lead" style="text-align: center;">Write one or two paragraphs describing your product, services or a specific feature.<br/> To be successful your content needs to be useful to your readers.</p>
-            <p style="text-align: center;">
-                <a href="/aboutus" class="btn btn-alpha">About us</a>
-                <a href="/contactus" class="btn btn-beta">Contact us</a>
-            </p>
+            <div class="s_cover_box text-center lead">
+                <h1><font style="font-size: 62px;">Catchy Headline</font></h1>
+                <p>Write one or two paragraphs describing your product, services or a specific feature.<br/> To be successful your content needs to be useful to your readers.</p>
+                <p><br/></p>
+                <p>
+                    <a href="/aboutus" class="btn btn-alpha">About us</a>
+                    <a href="/contactus" class="btn btn-beta">Contact us</a>
+                </p>
+            </div>
         </div>
     </section>
 </template>
 
+<template id="s_banner_options" inherit_id="website.snippet_options">
```

Comment:
Banner options in cover? + same comment as banner, put this in another PR @Cocographique

Reply:
**PR** [46922](https://github.com/odoo/odoo/pull/46922)